src/components/primitives/Slider/SliderTrack.tsx

```diff
@@ -12,7 +12,7 @@ const SliderTrack = ({ children, ...props }: ISliderProps) => {
     <Box
       position="absolute"
       backgroundColor={trackColor}
-      height={sliderSize}
+      height={`${sliderSize}px`}
```

Comment:
this is fixed now, can remove this. using size in number and px are two different things.